public/app/percona/check/Check.service.ts

```diff
@@ -0,0 +1,153 @@
+// @ts-nocheck
```

Comment:
Is it that bad that it needs `nocheck`?

Reply:
Thanks, checked and fixed TS, nocheck removed!